Volume/Views/Trending/TrendingFlyerCell.swift

```diff
@@ -0,0 +1,216 @@
+        static let categoryVerticalPadding: CGFloat = 4
+        static let dateFont: Font = .helveticaRegular(size: 12)
+        static let imageHeight: CGFloat = UIScreen.main.bounds.width - 32
+        static let locationFont: Font = .helveticaRegular(size: 12)
+        static let organizationNameFont: Font = .newYorkMedium(size: 10)
+        static let spacing: CGFloat = 4
+        static let titleFont: Font = .newYorkMedium(size: 20)
+    }
+    
+    // MARK: - UI
+    
+    var body: some View {
+        if let url = URL(string: flyer.pageURL) {
+            cellLinkView(url: url)
+        } else {
+            cellNoLinkView
+        }
+    }
+    
+    private func cellLinkView(url: URL) -> some View {
+        Link(destination: url) {
+            VStack(alignment: .leading, spacing: Constants.spacing) {
+                imageFrame
+                organizationName
+                flyerTitle
+                flyerDate
+                flyerLocation
+            }
+        }
+        .buttonStyle(EmptyButtonStyle())
+    }
+    
+    private var cellNoLinkView: some View {
+        VStack(alignment: .leading, spacing: Constants.spacing) {
+            imageFrame
+            organizationName
+            flyerTitle
+            flyerDate
+            flyerLocation
+        }
+    }
+    
+    private var imageFrame: some View {
+        ZStack(alignment: .topLeading) {
+            // TODO: Remove temporary image holder
+            if let flyerImage = urlImageModel.image {
+                ZStack(alignment: .center) {
+                    Color(uiColor: flyerImage.averageColor ?? .gray)
+                    
+                    Image(uiImage: urlImageModel.image ?? UIImage())
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .frame(height: Constants.imageHeight)
+                }
+            } else {
+                SkeletonView()
+            }
+            
+            Text(Organization.contentTypeString(
+                    // TODO: May need to change once backend implements
+                    type: flyer.organizations[0].contentType)
+                )
+                .padding(.init(
+                    top: Constants.categoryVerticalPadding,
+                    leading: Constants.categoryHorizontalPadding,
+                    bottom: Constants.categoryVerticalPadding,
+                    trailing: Constants.categoryHorizontalPadding
+                ))
+                .font(Constants.categoryFont)
+                .foregroundColor(Color.white)
+                .background(Color.volume.orange)
+                .clipShape(RoundedRectangle(cornerRadius: Constants.categoryCornerRadius))
+                .padding([.top, .leading], 8)
+                .frame(alignment: .topLeading)
+        }
+        .frame(height: Constants.imageHeight)
+    }
+    
+    private var bookmarkButton: some View {
+        Button {
+            Haptics.shared.play(.light)
```

Comment:
Not sure if you already did this, but since you are integrating haptics for these buttons, you might want to make sure that the bookmark, share, and other buttons on other scenes also mirror this behavior. You might also want to factor our this button implementation because I'm pretty sure it's being reused in other screens.

Reply:
This release won't have bookmarking/sharing since we are waiting for backend, but I will definitely write this down in my notes.